packages/parser-babel-config/tests/babel-config.ts

```diff
@@ -0,0 +1,141 @@
+import * as sinon from 'sinon';
+import test from 'ava';
+import { EventEmitter2 } from 'eventemitter2';
+import BabelConfigParser from '../src/babel-config';
+
+test.beforeEach((t) => {
+    t.context.sonarwhal = new EventEmitter2({
+        delimiter: '::',
+        maxListeners: 0,
+        wildcard: true
+    });
+});
+
+test.serial('If no file is parsed, it should emit a `notfound::babel-config` error', async (t) => {
+    const sandbox = sinon.sandbox.create();
```

Comment:
You don't have problems because you are stubbing a different object every time, but you should call to `sandbox.restore()` at the end of each tests.